TODO/explain-activity-launch-mode-with-examples.md

```diff
@@ -1,20 +1,20 @@
 > * 原文地址：[Explain Activity Launch Mode With Examples](http://www.songzhw.com/2016/08/09/explain-activity-launch-mode-with-examples/)
 * 原文作者：[songzhw](http://www.songzhw.com/author/songzhw2012gmail-com/)
 * 译文出自：[掘金翻译计划](https://github.com/xitu/gold-miner)
-* 译者： 
+* 译者： [Liz](http://lizwangying.github.io/)
 * 校对者： 
 
 ## adb shell dumpsys activity
 
-This command can give you a clear view of each task, like how many tasks do you have, which activities are in which task.
+输入这个命令可以得到一个清晰的 Task 视图，比如你有多少个 Task ，哪些 activity 在其对应的 Task 等相关信息。
 
-Now I have one picture of this command’s output.
+下图是一张运行这个命令的输出截图。
 
 ![](http://i2.wp.com/www.songzhw.com/wp-content/uploads/2016/08/20160214_01.png?w=644)
 
-From this output, we can see, now ,we have two Task (#103, #102).
+从图中可以看出，有两个 Task (#103, #102) 。
 
-Task #103 : affinity = “cn.six.task2”, size = 3 (it has three activities in it)
+Task #103 : affinity = “cn.six.task2”, size = 3 (它里面有三个activity)
 
 — Activity One    
 — Activity Three    
```

Comment:
Task #103 : affinity = “cn.six.task2”, size = 3 (it has three activities in it)
— Activity One 
— Activity Three
— ActivityTwo

Task #102 : affinity = “cn.six.adv”, size = 1
— Activity One